Gradle build fails with Could not resolve com.splunk.logging:splunk-library-javalogging:1.8.0 Received status code 401 from server: Unauthorized

### Description
As an developer, I want to be able to checkout the master branch and build the code. 

### Acceptance Criteria
* ./gradlew build should run without any issues

### Steps to Reproduce (Bug)
1. ./gradlew build

**Expected behavior:** Build to run fine

**Actual behavior:** Build fails with the following error while downloading dependencies
`> Task :startScripts FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':startScripts'.
> Could not resolve all files for configuration ':runtimeClasspath'.
   > Could not resolve com.splunk.logging:splunk-library-javalogging:1.8.0.
     Required by:
         project : > project :besu
      > Could not resolve com.splunk.logging:splunk-library-javalogging:1.8.0.
         > Could not get resource 'https://repo.spring.io/libs-release/com/splunk/logging/splunk-library-javalogging/1.8.0/splunk-library-javalogging-1.8.0.pom'.
            > Could not GET 'https://repo.spring.io/libs-release/com/splunk/logging/splunk-library-javalogging/1.8.0/splunk-library-javalogging-1.8.0.pom'. Received status code 401 from server: Unauthorized
`

**Frequency:** 100%

--- a/build.gradle
+++ b/build.gradle
@@ -126,6 +126,7 @@ allprojects {
     maven { url "https://hyperledger-org.bintray.com/besu-repo" }
     maven { url "https://consensys.bintray.com/pegasys-repo" }
     maven { url "https://consensys.bintray.com/consensys" }
+    maven { url "https://splunk.jfrog.io/splunk/ext-releases-local" }
     maven { url "https://dl.bintray.com/open-telemetry/maven" }
   }
 

--- a/build.gradle
+++ b/build.gradle
@@ -126,7 +126,6 @@ allprojects {
     maven { url "https://hyperledger-org.bintray.com/besu-repo" }
     maven { url "https://consensys.bintray.com/pegasys-repo" }
     maven { url "https://consensys.bintray.com/consensys" }
-    maven { url "https://repo.spring.io/libs-release" }
     maven { url "https://dl.bintray.com/open-telemetry/maven" }
   }
 

